packages/runtime/src/widgets/Form/MultiSelect.tsx

```diff
@@ -40,6 +40,7 @@ export type MultiSelectProps = {
   items?: Expression<SelectOption[]>;
   onItemSelect?: EnsembleAction;
   hintStyle?: EnsembleWidgetStyles;
+  mode?: "multiple" | "tags";
```

Comment:
Pick a better attribute name - it needs to be intuitive to the end user not just a passthrough of whatever internal component you are using. That is a poor abstraction.